### PR TITLE
Revised text on persistence

### PR DESCRIPTION
**Pull request by jandrieu**
This PR moves the section on persistence to the security consideration section and rewrites it to address issues discussed on multiple special topic and regular calls.

I've removed the language about "permanent binding" and discussed the security implications of depending on persistence.

This replaces PR #457

***
<a href="https://pr-preview.s3.amazonaws.com/jandrieu/did-core/pull/574.html" title="Last updated on Feb 2, 2021, 1:50 PM UTC (0fb160d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/574/518362a...jandrieu:0fb160d.html" title="Last updated on Feb 2, 2021, 1:50 PM UTC (0fb160d)">Diff</a>

### CODE REVIEW

**Review by ChristopherA — APPROVED** [review]
+1

**Review by rhiaro — APPROVED** [review]

**Comment by jandrieu** [comment]
@rhiaro and @ChristopherA please take a look again. I realized after submitting the initial draft that the text focused a bit too much on ambiguity from proof-of-control without discussing the ambiguity from semantic drift.

**Review by ChristopherA — COMMENTED** [review]
Typo: c/thenm/them/

**Review by ChristopherA — APPROVED** [review]
+1 (other than minor typo)

**Comment by jandrieu** [comment]
Ok. Typo fixed.

File: index.html
Comment (dlongley) [inline index.html:4839]:
```suggestion
that same DID, because to <strong>them</strong> the DID refers to the 
```

File: index.html
Comment (dlongley) [inline index.html:4837]:
```suggestion
that identifier referred to the business. As the business grows it
```

File: index.html
Comment (dlongley) [inline index.html:4815]: Needs line breaks.

File: index.html
Comment (dlongley) [inline index.html:4830]:
```suggestion
technically possible for the controller to reuse a DID for different
```

File: index.html
Comment (dlongley) [inline index.html:4801]: I'm in general agreement with this paragraph, but since controllers can delegate control, I think this should be slightly tweaked. I think what's important here is that DIDs are designed to enable independent control by controllers, i.e., you get full control over your identifiers (including the ability to delegate control as you see fit), you don't just rent them from a service that has essentially delegated control to *you*, that they can later revoke as they see fit.

```suggestion
DIDs are designed to be persistent such that a controller need not rely
upon a single trusted third party or administrator to maintain their
identifiers. No administrator can take control away from the controller,
nor can an administrator prevent their identifiers' use for any particular
purpose such as authentication, authorization, and attestation. No
third party can act on behalf of a controller to remove or render
inoperable an individual's (or an organization's) identifier without the
controller's consent.
```

**Review by dlongley — APPROVED** [review]
Approving with suggestions.

**Comment by ChristopherA** [comment]
@dlongley wrote:
> what's important here is that DIDs are designed to enable independent control by controllers, i.e., you get full control over your identifiers (including the ability to delegate control as you see fit), you don't just rent them from a service that has essentially delegated control to you, that they can later revoke as they see fit.

I like the clarity of this text. Tagging it for reuse elsewhere.

File: index.html
Comment (jandrieu) [inline index.html:4816]:
```suggestion
mechanism has been compromised. It may well be that the expected controller
still has access to the private keys—and as such can execute a
```

File: index.html
Comment (jandrieu) [inline index.html:4818]:
```suggestion
proof-of-control as part of a verification process—while at the same time, 
a bad actor also has access to (or a copy of) those same keys.
```

File: index.html
Comment (jandrieu) [reply index.html:4841]:
```suggestion
that identifier referred to the business. As the business grows, it
```

File: index.html
Comment (brentzundel) [inline index.html:4853]:
```suggestion
imply that they refer to the exact same Subject, nor that they are under the
```

**Review by brentzundel — APPROVED** [review]
Approved with mine and Dave's suggestions

**Comment by talltree** [comment]
@jandrieu Your PR takes a different approach that I took with the rewrite of #457, but overall I am good with it. The one thing I feel is missing is that it doesn't mention RFC 8141 or URNs at all. I feel strongly that any discussion of persistence in identifiers needs to explain the relationship to 8141—without changing any of the points you make in your PR.

Are you open to me suggesting text that would address my concern? If so, I'll strive to do that tomorrow (Tuesday), and if that text is agreeable, we can close #457 and proceed with this.

**Review by kdenhartog — APPROVED** [review]
I can live with the direction this PR goes. It doesn't preclude my original concerns about resource based identifiers (and opts to leave that as tribal knowledge which I think is fine), but does address your concern about the semantic ambiguity of a DID that identifies a non digital entity.

File: index.html
Comment (TallTed) [inline index.html:4818]:
```suggestion
still has access to the private keys &mdash; and as such can execute a
proof-of-control as part of a verification process &mdash; while at the same time, 
```

**Comment by jandrieu** [comment]
@talltree I don't see how URNs have anything to do with the issues raised here.

**Comment by talltree** [comment]
> @talltree I don't see how URNs have anything to do with the issues raised here.

@jandrieu After studying your PR closely, I agree. In fact I realized that the issues I wanted to see addressed about using DIDs as URNs was the primary purpose of my PR #457. So I'm simply revising that PR to address that specific issue as its own subsection of Security Considerations.

So for this PR I have just one tiny formatting/ordering suggestion which I'll make separately.

File: index.html
Comment (talltree) [inline index.html:4795]: Joe, for some reason when I preview this PR, this section appears as a subsection of the previous section instead of a sibling to it. But that might be a preview bug because the section tag appears correct.

In any case, you might also consider making it section 9.9, i.e., right after 9.8 "The Role of Human-Friendly Identifiers", as it seems to fit right before "Immutability of DID Documents".

**Review by talltree — APPROVED** [review]
Joe, see my one comment about the order where this subsection appears inside the Security Considerations section.

File: index.html
Comment (jandrieu) [reply index.html]: Good catch. There was a </section> tag in the wrong place.

File: index.html
Comment (msporny) [inline index.html:4816]:
```suggestion
mechanism has been compromised. It might well be that the expected controller
```

File: index.html
Comment (msporny) [inline index.html:4822]:
```suggestion
As such, cryptographic proof-of-control is expected to only be used as one
```

File: index.html
Comment (msporny) [inline index.html:4854]:
```suggestion
control of the same controller. Instead, one needs to understand the context
```

**Review by msporny — APPROVED** [review]
Minor nits, other than that, LGTM.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2019/did-wg/Meetings/Minutes/2021-01-28-did-topic#section1) on 2021-01-28

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

### 1. PRs on the Appendix 

_See github pull request [#457](https://github.com/w3c/did-core/pull/457), [#460](https://github.com/w3c/did-core/pull/460), [#574](https://github.com/w3c/did-core/pull/574)._




**Manu Sporny:** Real quick, Drummond, just to check in on your PRs. For appendix and persistence and the URN thing. Can we get an update on those?  

**Drummond Reed:** Lots of progress on one, not the other. Came down to the amount of time in the day. The PR on persistence... on our last call, I said "Hey, Joe, can I add something on URNs." He said he'd consider it.  
… When I looked at his PR it was really about a different set of topics on persistence. In the end I felt that the points I wanted to be made about URNs, and so on should just be in a different section, which I called DIDs as URNs.  
… Rather than shoehorn/appending it to Joe's section, I just made a new section. I revised it to just be a standalone section with a different name. Meanwhile, I reviewed Joe's PR and made one suggestion about formatting. That's my full report. Because I changed the substance of what my PR was suggesting, I just went back to the reviewers to ask to look at it again.  
… I completed last night at midnight so maybe no reviews yet.  

**Manu Sporny:** I'm hearing persistence PR is on a good glide path, good reviews, will probably go in. You've got another PR that needs review, and maybe there's alignment there with some of Joe's comments on it. Potential alignment there. There were a lot of change suggestions for the appending, but it seems no one is objecting anymore.  

**Drummond Reed:** Everything after Joe had done the same thing and read it closely, he's got a particular way to characterize identification and in the end we agreed. There are like 20 small suggestions, I agree with all of them and just need time to incorporate them. The largest one is a formatting suggestion from Ivan and I just need to reformat to be a single section.  
… Then I need to change the settings inside and do wordsmithing.  
… My next highest priority is to get that done by the end of this weekend. My next assignment after that is either another section or adding more to a section on privacy regulations. I've prepared wording for that, short and sweet.  
… Then I can put all my energy into review.  
… That's my report.  

**Manu Sporny:** Unless there are any objections, we can move onto concerns around the CBOR section. Anyone want to cover anything else first?  

</details>

**Comment by msporny** [comment]
Waiting on @jandrieu to merge suggestions and fix merge conflicts.

**Comment by jandrieu** [comment]
Suggestions adopted. Merge conflicts resolved.

This should be ready to merge.

File: index.html
Comment (TallTed) [inline index.html:5127]:
```suggestion
<em>contextually</em> rather than absolutely. Their persistence does not 
```

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**
